### PR TITLE
[LEARNER-325] update the checkout page with the new design

### PR DESCRIPTION
**Pull request by marjev**
Review:
@edx/helio 

JIRA ticket:
[update the checkout page with the new design](https://openedx.atlassian.net/browse/LEARNER-325)

Screenshot:
screencapture-localhost-8002-basket-1491393194011

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @marjev! It looks like you're a member of a company that does contract work for edX. If you're doing this work as part of a paid contract with edX, you should talk to edX about who will review this pull request. If this work is not part of a paid contract with edX, then you should ensure that there is an OSPR issue to track this work in [JIRA](https://openedx.atlassian.net), so that we don't lose track of your pull request.

<a   href="https://openedx-webhooks.herokuapp.com/github/process_pr?repo=edx%2Fecommerce&number=1237"   rel="nofollow" >Create an OSPR issue for this pull request</a>.

**Review by vkaracic — APPROVED** [review]
Include a screenshot.

**Comment by marjev** [comment]
I've added a screenshot.

File: ecommerce/templates/oscar/basket/partials/client_side_checkout_basket.html [hunk 1]
```diff
@@ -13,6 +13,7 @@
 
         <div role="region" aria-labelledby="cart-region">
             <h2 id="cart-region" class="title">{% trans "in your cart" %}</h2>
+            <p>{% trans "Your purchase contains the following:" %}</p>
```
Comment (clintonb) [inline ecommerce/templates/oscar/basket/partials/client_side_checkout_basket.html]: Take the semi-colon out of the `trans` tag.

**Review by clintonb — APPROVED** [review]

**Pull request merged**